Conceptpower+Spring/src/main/webapp/WEB-INF/spring/spring-security.xml

```diff
@@ -15,19 +15,25 @@
 		<logout
 		  logout-success-url="/"
 		  logout-url="/signout" delete-cookies="JSESSIONID" />
-		
-		<intercept-url pattern="/welcome*" access="permitAll" />
-
-		<intercept-url pattern="/resources/**" access="permitAll" />
+		  
+		<intercept-url pattern="/conceptpower/rest/concept/add" access="isAuthenticated()" />
 
 		<intercept-url pattern="/auth/user/**" access="hasRole('ROLE_CP_ADMIN')" />
 		<intercept-url pattern="/auth/index/**" access="hasRole('ROLE_CP_ADMIN')" />
+		<intercept-url pattern="/auth/conceptlist/**" access="isAuthenticated()" />
+		<intercept-url pattern="/auth/concepttype/**" access="isAuthenticated()" />	
```

Comment:
are those two actually neccessary? aren't they covered by line 26?

Reply:
Oh yes, sorry about that. Fixed this and I also rearranged the intercept-urls access-level wise